modules/xmpp/BreakoutRooms.js

```diff
@@ -7,6 +7,7 @@ const FEATURE_KEY = 'features/breakout-rooms';
 const BREAKOUT_ROOM_ACTIONS = {
     ADD: `${FEATURE_KEY}/add`,
     MOVE_TO_ROOM: `${FEATURE_KEY}/move-to-room`,
+    MESSAGE_TO_ROOM: `${FEATURE_KEY}/message-to-room`,
```

Comment:
You need to have the prosody plugin handle these messages, but I haven't seen such a PR, are you still working on it?

Reply:
realizing that i dont even know how prosody handles regular messages, I didnt knew where to start. 
Could you point me to the direction and i'll see if i'm able to create one ? 

i guess in this case until I added a PR for prosody ( in jitsi-meet ? or is it a different repo i didnt find yet ? ) i'll close this again, until its relevant.

Reply:
Feel free to leave it open, perhaps better mark it as a daft. Here is the BR plugin: https://github.com/jitsi/jitsi-meet/blob/master/resources/prosody-plugins/mod_muc_breakout_rooms.lua you can take a look at how other messages are handled.

Reply:
Thanks, marked it as draft. 

I did find the breakout plugin, but its not really easy to understand how the Lua plugins interact with prosody main code. 

At least there's no plugin for the chat messages and other Basic xmpp functions. 
Or I just didn't find it.  

I hope it's not buried in the java code, I certainly didn't wanted to touch that. Even if it's just for understanding purposes.

Reply:
Chat is plain XMPP, but you need extra functionality, hence the need for a plugin.

Reply:
Gotta be honest here.  Lua is definitely not my main language, and its absolutely not clear how the objects are structured. Or at least I was unable to figure out where to look it up. 
I gathered as much references from other prosody plugins, as well as third party prosody plugins as I was able to find. 
PR is open. (https://github.com/jitsi/jitsi-meet/pull/13745)

I'll also open this one again.